template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/views/RdsView.java

```diff
@@ -1,11 +1,17 @@
 package com.sequenceiq.cloudbreak.template.views;
 
+import java.util.Objects;
+
+import javax.annotation.Nonnull;
```

Comment:
We might not need this.

Reply:
Just a syntactic sugar; it does not hurt but has no visible effect either. I did not want to use the similar one from JetBrains as this is is at least based on a JSR (even if basically abandoned).